crates/constraint-vm/src/error.rs

```diff
@@ -293,6 +293,12 @@ pub enum TotalControlFlowError {
     /// Attempted to halt if with an invalid condition
     #[error("halt if requires a boolean condition")]
     InvalidHaltIfCondition,
+    /// Attempted to panic if with an invalid condition
+    #[error("panic if requires a boolean condition")]
+    InvalidPanicIfCondition,
+    /// The `PanicIf` operation was called with a `true` argument
+    #[error("program panicked with `PanicIf` operation. The stack at the time of panic: {0:?}")]
+    Panic(Vec<Word>),
```

Comment:
I wonder if instead, we should just always return the stack in the `ConstraintError` type, rather than only when we panic :thinking: Probably fine like this for now.

Reply:
Yeh we could do. Might leave it to a follow up though l. I basically added it to make panic better for debugging